internal/painter/image.go

```diff
@@ -22,7 +22,7 @@ import (
 // PaintImage renders a given fyne Image to a Go standard image
 func PaintImage(img *canvas.Image, c fyne.Canvas, width, height int) image.Image {
 	if width <= 0 || height <= 0 {
-		return nil
+		return image.NewNRGBA(image.Rect(0, 0, 0, 0))
```

Comment:
Is this required for cardcontainer? Seem unnecessary to allocate this. Can't the caller do a nil check?

Reply:
removed